frontend/src/components/project_page/components/MemberList.tsx

```diff
@@ -34,7 +34,7 @@ const MemberList: FunctionComponent<MemberListProps> = ({ projectId }) => {
   }
 
   return (
-    <div className="flex ">
+    <div className="flex -space-x-4">
```

Comment:
if this `-space-x-4` isn't used, it's better to omit it from the changes in this PR

Reply:
the `-space-x-4` is creating the overlapping effect on user avatars

Reply:
ahh i see, that makes sense then